tests/unit/test_utils.py

```diff
@@ -409,3 +409,20 @@ def test_can_escape_recursive_shapes(self):
         }
         self.walker.walk(self.get_shape_model('Recursive'), self.visitor)
         self.assert_visited_shapes(['Recursive'])
+
+
+class TestStreamingBlob(unittest.TestCase):
```

Comment:
Instead of relying on `unittest` for test cases declaration, let's convert this to `pytest` style meaning:

1. No subclassing from `unittest.TestCase`
2. Using fixtures instead of the `setUp` method: https://docs.pytest.org/en/6.2.x/fixture.html. We'd likely would want just a fixture for a fake `argument_model` and then we can set the necessary properties (e.g. `type_name` and `serilization`) in each respective test case.
3. Using `assert` instead of methods from `unittest.TestCase`

Most of the codebase is still relying on `unittest` style but in the long run we want test to be following more closely to pytest style of writing test, as it is the standard now days. So for any new tests, we've been trying to use pytest style of tests assuming it would be straightforward/possible. I would recommend checking out the botocore codebase for  examples of how tests are written using pytest style.